internal/util/debug/debug.go

```diff
@@ -88,6 +91,29 @@ func Listen(opts *ListenOpts) (*Handler, error) {
 
 	stdL := must.NotFail(zap.NewStdLogAt(opts.L, zap.WarnLevel))
 
+	archiveDurations := prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "archive_response_seconds",
+			Help:      "Archive response time seconds.",
+			Buckets:   []float64{0.1, 0.5, 1, 5},
+		},
+		[]string{"code"},
+	)
```

Comment:
I don't see how that would be useful given that amount of information in the archive will change over time